package.json

```diff
@@ -418,7 +418,7 @@
     "scripts": {
         "completeClean": "rimraf node_modules/ .vscode-test/ dist/ server/dist/ server/node_modules/ server/out/ mlxprs-${version}.vsix",
         "npmInstallClientAndServer": "npm install && cd server && npm install && cd ..",
-        "testAll": "npm run test && npm run testIntegration && npm run testServer",
+        "testAll": "npm run test && npm run testServer",
```

Comment:
Does `testIntegration` no longer need to be run?

Reply:
It does not. In fact, testIntegration does not work work properly.